src/Illuminate/Http/Client/RequestException.php

```diff
@@ -13,6 +13,13 @@ class RequestException extends HttpClientException
      */
     public $response;
 
+    /**
+     * The truncation length for the exception message.
+     *
+     * @var int|false
+     */
+    public static $truncateAt = 120;
+
```

Comment:
Since this property has setters, is there the need to read the property as well since you made it public?

Reply:
This is common in Laravel and in it's first-party packages. See Cashier for example:

https://github.com/laravel/cashier-stripe/blob/15.x/src/Cashier.php

Reply:
What are the reasons for this?

Reply:
I think to provide the ability for developers to use whichever method of setting the static they prefer. Don't quote me on that though.

Reply:
I haven't looked at the code, but I assume, Cashier then forgot to add this "flexibility" to countless places